### PR TITLE
test: refactor smoketests

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
test, refactor

**Did you add tests for your changes?**
yes
**If relevant, did you update the documentation?**
No
**Summary**

Refactor code, follow DRY code.

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by alexander-akait — APPROVED** [review]
/cc @webpack/cli-team

**Review by anshumanv — DISMISSED** [review]

File: smoketests/helpers.js [hunk 1]
```diff
@@ -17,7 +21,121 @@ const swapPkgName = (current, isSubPackage = false) => {
     fs.renameSync(getPkgPath(current, isSubPackage), getPkgPath(next, isSubPackage));
 };
 
+const CLI_ENTRY_PATH = getBinPath();
```
Comment (jamesgeorge007) [inline smoketests/helpers.js:24]:
```suggestion
const CLI_ENTRY_PATH = path.resolve(ROOT_PATH, './packages/webpack-cli/bin/cli.js');;
```

`getBinPath()` isn't used anywhere else.

Comment (snitin315) [reply smoketests/helpers.js]: Done 👍🏻

Comment (jamesgeorge007) [reply smoketests/helpers.js]: Same as the case with `swapPkgName()`.

**Review by alexander-akait — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]

File: smoketests/helpers.js (hunk 1 above)
Comment (snitin315) [reply smoketests/helpers.js]: `swapPkgName()` is used multiple times in this file.

Comment (jamesgeorge007) [reply smoketests/helpers.js]: I mean, It need not be exported.

Comment (snitin315) [reply smoketests/helpers.js]: updated 👍🏻

**Review by jamesgeorge007 — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]
lgtm

**Pull request merged**